src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -104,10 +105,16 @@ double kerr_schild_areal_radius_from_isotropic(const double isotropic_radius,
 
 DataVector kerr_schild_areal_radius_from_isotropic(
     const DataVector& isotropic_radius, const double mass) {
-  return RootFinder::toms748(
-      [&isotropic_radius, &mass](const double areal_radius, const size_t i) {
-        return kerr_schild_isotropic_radius_from_areal(areal_radius, mass) -
-               isotropic_radius[i];
+  return RootFinder::toms748<true>(
```

Comment:
Question: is this a test if the vectorization is working? Any reason you chose this particular function? Are you planning to go through and do some profiling and vectorize other toms748 calls in the code?

Reply:
1. More or less. I just wanted to make sure it works on code other than the stuff I've already done and this seemed like an easy candidate.
2. This seemed like an easy candidate.
3. Sort of. FixConservatives and primitive recovery yes, beyond that, no unless it ends up being something that slows me down.